content/calico/stars_policy_demo/create_resources.md

```diff
@@ -33,43 +39,52 @@ kubectl apply -f namespace.yaml
 We will create frontend and backend [replication controllers](https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/) and [services](https://kubernetes.io/docs/concepts/services-networking/service/) in this namespace in later steps.
 
 
-Create a new file called `management-ui.yaml` with the following contents:
+Copy/Paste the following commands into your Cloud9 Terminal.
+```
+cd ~/environment/calico_resources
```

Comment:
You can remove this command (seems redundant from previous command)
cd ~/environment/calico_resources

Reply:
I'm leaving it in there, because it's harmless, and I want to make sure we get the file in the right place.